search/Sta.cc

```diff
@@ -2953,7 +2953,7 @@ Sta::netSlack(const Net *net,
 {
   ensureGraph();
   Slack slack = MinMax::min()->initValue();
-  NetConnectedPinIterator *pin_iter = network_->connectedPinIterator(net);
+  std::unique_ptr<NetConnectedPinIterator> pin_iter(network_->connectedPinIterator(net));
```

Comment:
You may notices the absence of unique_ptr in OpenSTA. That is not an accident.
Please just add the the delete like every other call to a network iterator.